src/test/kotlin/minesweeper/domain/MinesTest.kt

```diff
@@ -0,0 +1,50 @@
+package minesweeper.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class MinesTest {
+
+    private lateinit var mines: Mines
+    private lateinit var minesExpected: List<Mine>
+
+    @BeforeEach
+    fun setUp() {
+        mines = Mines(listOf(Pair(0, 0), Pair(1, 1), Pair(0, 0)).map { Mine(it) })
+        minesExpected = listOf(Pair(0, 0), Pair(1, 1)).map { Mine(it) }
+    }
+
+    @Test
+    fun `중복을 검사한다`() {
+        assertTrue(mines.hasDuplicate())
```

Comment:
AssertJ의 `Assertions`를 사용하면 어떨까요? 아래의 글을 참고하세요.
https://www.baeldung.com/introduction-to-assertj

```suggestion
        assertThat(mines.hasDuplicate()).isTrue()
```